crates/libcontainer/src/container/init_builder.rs

```diff
@@ -1,5 +1,5 @@
 use anyhow::{bail, Context, Result};
-use nix::unistd;
+use nix::unistd::{self};
```

Comment:
Unnecessary...?

Reply:
Aha, yes. I am surprised that clippy was not triggered on this.